packages/plugins/IconMenu/README.md

```diff
@@ -11,40 +11,70 @@ _Please note that the average user has never seen the client before and just wan
 
 ## Configuration
 
+### iconMenu
+
 > ⚠️ This plugin can't be configured with the `mapConfiguration`, but is configured during client construction. Currently, no way exists to add plugins to it after the build took place.
 
 For details on the `displayComponent` attribute, refer to the [Global Plugin Parameters](../../core/README.md#global-plugin-parameters) section of `@polar/core`.
 
-Usage during client build:
+| fieldName | type | description |
+| - | - | - |
+| initiallyOpen  | string?   | Id of the plugin which should be open on start; only applicable if the device doesn't have a small display  |
+| menus | Array | Defines which plugins should be rendered as part of the icon menu. |
```

Comment:
While human-readable, we're keeping our types close to how they're working in the Masterportal so that, if the need ever arises, we'll be able to have an Admintool in the same fashion as it exists there. For this to work, the type needs a little change.

```suggestion
| menus | menuEntry[] | Defines which plugins should be rendered as part of the icon menu. |
```

Further down, the chapter `### iconMenu.menus` is then renamed to `### iconMenu.menuEntry`. I originally thought it should be renamed to `### iconMenu.menu`, but just now realized that we've got the names all wrong. After all, we produce a single menu with entries, and not multiple menus in a menu of menus ... but I guess that's how it is for now and we should avoid the breaking change of renaming the above `menus` to `menu`.

Now, I wonder whether the paragraph above is human-readable. 😄

Reply:
f536941285d77b6098b3d622547a8a24d77d1db0